recipes/polyhedral-gravity-model/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "polyhedral-gravity-model" %}
+{% set org = "esa" %}
+{% set version = "1.1.0" %}
+{% set sha256 = "2255b2cd136cfd0e7f02bb1020f67af4ca2c99e23e4006e18077e10dc0a326ac" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ org }}/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  script_env:
+    - CMAKE_GENERATOR=Ninja
+    - PARALLELIZATION_DEVICE=OMP
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [linux]
+    - ninja
+    - python
```

Comment:
Sorry I missed that before, but:
- `python` shouldn't be listed in build. Only in host.
- `pybind11` should be in host (not build). Same for `spdlog`, `yaml-cpp`, `xsimd`.

The `build` sections is only for build tools like compilers, make... Libraries used during the build process should be in `hosts` (`llvm-openmp` and `libgomp` should stay in build).

Reply:
"automatic update" of `sha256` definitely convinces me :) Thanks for the recommendation @beenje 

As suggested, the libraries are also moved from the `build` to the `host` section.